app/src/shared/components/SelectInput/index.jsx

```diff
@@ -0,0 +1,105 @@
+// @flow
+
+import React from 'react'
+import Select, { components } from 'react-select'
+
+import FormControl, { type FormControlProps } from '../FormControl'
+import TickIcon from '../TickIcon'
+import styles from './selectInput.pcss'
+
+type Props = FormControlProps & {
+    placeholder?: string,
+    options: Array<any>,
+    value: any,
+    name: string,
+    onChange: Function,
+    required: boolean,
+    clearable: boolean,
+}
+
+const customStyles = {
```

Comment:
I don't believe they actually work... the inline styles takes preference over the classnames. The other way to do it would be to define custom components, set `getStyles` as a no-op and define styles by classes. Left it as it is now :)